distributor/Dockerfile

```
@@ -10,4 +10,12 @@ ADD . /usr/src/
 
 RUN ballerina build --skip-tests distributor
 
-CMD ballerina run target/bin/distributor.jar --eclk.hub.db.url=jdbc:mysql://$DB_URL --eclk.hub.db.username=$DB_UN --eclk.hub.db.password=$DB_PW --eclk.hub.db.useSsl=$DB_USE_SSL --eclk.hub.db.encryptionkey=$DB_ENCRYPTION_KEY --eclk.sms.twilio.accountSid=$TWILIO_SID --eclk.sms.twilio.authToken=$TWILIO_AUTHTOKEN --eclk.sms.twilio.source=$TWILIO_SOURCE
+FROM openjdk:8
+
+ENV TZ=Asia/Colombo
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+COPY  --from=build /usr/src/target /usr/src/target
+WORKDIR /usr/src/
+
+CMD java -jar target/bin/distributor.jar --eclk.hub.db.url=jdbc:mysql://$DB_URL --eclk.hub.db.username=$DB_UN --eclk.hub.db.password=$DB_PW --eclk.hub.db.useSsl=$DB_USE_SSL --eclk.hub.db.encryptionkey=$DB_ENCRYPTION_KEY --eclk.sms.twilio.accountSid=$TWILIO_SID --eclk.sms.twilio.authToken=$TWILIO_AUTHTOKEN --eclk.sms.twilio.source=$TWILIO_SOURCE
```

Since we have moved from twilio to GovSMS client, I changed the credentials

```suggestion
CMD java -jar target/bin/distributor.jar --eclk.hub.db.url=jdbc:mysql://$DB_URL --eclk.hub.db.username=$DB_UN --eclk.hub.db.password=$DB_PW --eclk.hub.db.useSsl=$DB_USE_SSL --eclk.hub.db.encryptionkey=$DB_ENCRYPTION_KEY --eclk.govsms.username=$GOVSMS_UN --eclk.govsms.password=$GOVSMS_PW --eclk.govsms.source=$GOVSMS_SOURCE
```